components/camera/platforms/jetson/camera.go

```diff
@@ -1,66 +1,18 @@
 package jetsoncamera
 
-// #include <sys/utsname.h>
```

Comment:
@seanavery This was really helpful so I moved it to the utils folder to generalize it. No functionality was changed.